go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -756,7 +756,7 @@ func (npw *nodePortWatcher) SyncServices(services []interface{}) error {
 				}
 			}
 
-			keepIPTRules = append(keepIPTRules, egressSVCIPTRulesForEndpoints(service, v4Eps.UnsortedList(), v6Eps.UnsortedList())...)
```

Comment:
so sorting involves some extra computation, so we don't want to change all UnsortedList() to List(), we should do it only in the places where List() was used before the kube rebase bump was done... example in this place it was always UnsortedList() that was used and so we don't need to change it?

Reply:
makes sense, I should have checked that, will go through and revert where that does not need to be